optimism-mainnet.json

```diff
@@ -7,452 +7,732 @@
-      "logoURI": "https://ethereum-optimism.github.io/data/wstETH/logo.svg",
-      "extensions": {
-        "baseBridgeAddress": "0xac9D11cD4D7eF6e54F14643a393F68Ca014287AB",
-        "opListId": "default",
-        "opTokenId": "wstETH"
-      }
-    },
-    {
-      "chainId": 1,
-      "address": "0x01ba67aac7f75f647d94220cc98fb30fcc5105bf",
-      "name": "Lyra",
-      "symbol": "LYRA",
-      "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/data/LYRA/logo.png",
-      "extensions": {
-        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1",
-        "opListId": "default",
-        "opTokenId": "LYRA"
-      }
-    },
-    {
-      "chainId": 10,
-      "address": "0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb",
-      "name": "Lyra",
-      "symbol": "LYRA",
-      "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/data/LYRA/logo.png",
-      "extensions": {
-        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010",
-        "opListId": "default",
-        "opTokenId": "LYRA"
-      }
-    },
-    {
-      "chainId": 1,
-      "address": "0x431ad2ff6a9C365805eBaD47Ee021148d6f7DBe0",
-      "name": "dForce",
-      "symbol": "DF",
-      "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/data/DF/logo.svg",
-      "extensions": {
-        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1",
-        "opListId": "extended",
-        "opTokenId": "DF"
-      }
-    },
-    {
-      "chainId": 10,
-      "address": "0x9e5AAC1Ba1a2e6aEd6b32689DFcF62A509Ca96f3",
-      "name": "dForce",
-      "symbol": "DF",
-      "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/data/DF/logo.svg",
-      "extensions": {
-        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010",
-        "opListId": "extended",
-        "opTokenId": "DF"
-      }
+        "chainId": 10,
+        "address": "0xcb59a0a753fdb7491d5f3d794316f1ade197b21e",
+        "name": "TrueUSD",
+        "symbol": "TUSD",
+        "decimals": 18,
+        "logoURI": "https://optimistic.etherscan.io/token/images/bridgedtrueusd_32.png"
+    },
+    {
+        "chainId": 10,
+        "address": "0x94b008aa00579c1307b0ef2c499ad98a8ce58e58",
+        "name": "Tether USD",
+        "symbol": "USDT",
+        "decimals": 6,
+        "logoURI": "https://optimistic.etherscan.io/token/images/tether_32.png"
+    },
+    {
+        "chainId": 10,
+        "address": "0x7f5c764cbc14f9669b88837ca1490cca17c31607",
+        "name": "USD Coin",
+        "symbol": "USDC",
+        "decimals": 6,
+        "logoURI": "https://optimistic.etherscan.io/token/images/usdc_32.png"
+    },
+    {
+        "chainId": 10,
+        "address": "0x0b2c639c533813f4aa9d7837caf62653d097ff85",
+        "name": "USD Coin",
+        "symbol": "USDC",
+        "decimals": 6,
+        "logoURI": "https://optimistic.etherscan.io/token/images/usdc_32.png"
+    },
```

Comment:
We are adding twice `USDC` with a different contract address, is this expected? @caioflores

Reply:
Yes, one is Bridged USDC, and the other on is not.